source/_components/light.plum_lightpad.markdown

```diff
@@ -0,0 +1,26 @@
+---
+layout: page
+title: "Plum Lightpad Light Switch"
+description: "Instructions how to integrate Plum Lightpads into Home Assistant."
+date: 2017-11-20 00:45
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: plum-lightpad.png
+ha_category: Light
+ha_iot_class: "Local Push"
+---
```

Comment:
Missing version tag

Reply:
What is the next release? I'm guessing: 0.58.2?

Reply:
`0.59`, new features are never added in minor releases.